e107_plugins/hero/languages/English/English_admin.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/*
++----------------------------------------------------------------------+
+|        e107 website content management system Hungarian Language File
+|        Released under the terms and conditions of the
+|        GNU General Public License (http://gnu.org).
+|        Last Modified: 2020/12/18 12:42:44
+|
+|        $Author: Yesszus $
++----------------------------------------------------------------------+
+*/
+
+define("LAN_HERO_SUCC", "Default slides imported.");
```

Comment:
@yesszus Thank you for this! It's great, it just needs to use the correct naming standard. 

```
LAN_HERO_ADMIN_001
LAN_HERO_ADMIN_002
...

```